eng/tools/gen-sdk/.prettierignore

```diff
@@ -0,0 +1,4 @@
+*.json
+*.md
+*.jsonc
```

Comment:
Lets try to avoid having duplicate copies of these files

Reply:
Removed